CONTRIBUTING.md

```diff
@@ -0,0 +1,374 @@
+# Contributing to Janus IDP's Backstage plugins collection
+
+✨ We would love for you to contribute to Janus IDP collection of Backstage plugins and help make it even better than it is today! ✨
+
+As a contributor, here are the guidelines we would like you to follow:
+
+- [Code of conduct](#code-of-conduct)
+- [How can I contribute?](#how-can-i-contribute)
+- [Using the issue tracker](#using-the-issue-tracker)
+- [Submitting a Pull Request](#submitting-a-pull-request)
+- [Coding rules](#coding-rules)
+- [Working with the code](#working-with-the-code)
+
+We also recommend that you read [How to Contribute to Open Source](https://opensource.guide/how-to-contribute).
+
+## Code of conduct
+
+Help us keep **Janus-IDP** open and inclusive. Please read and follow our [Code of Conduct](CODE_OF_CONDUCT.md).
+
+## How can I contribute?
+
+### Improve documentation
+
+As a **Janus-IDP** user, you are the perfect candidate to help us improve our documentation: typo corrections, clarifications, more examples, etc. Take a look at the [documentation issues that need help](https://github.com/janus-idp/backstage-plugins/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3Akind%2Fdocumentation).
+
+Please follow the [Documentation guidelines](#documentation).
+
+### Give feedback on issues
+
+Some issues are created without the information requested in the [Bug report guideline](#bug-report).
```

Comment:
What about a sentence more like: "Issues that lack relevant information can be very difficult to track down and fix, please follow the [Bug reporting guidelines](#bug-report) to help make them easier to resolve."?